tests/dummy/lib/ember-blog/addon/controllers/post.js

```diff
@@ -1,5 +1,14 @@
 import Controller from '@ember/controller';
+import { set } from '@ember/object';
 
 export default Controller.extend({
   queryParams: ['lang'],
+
+  actions: {
+    goHomeProgrammatically() {
```

Comment:
```suggestion
    transitionToHomeFromController() {
```

We need to distinguish this action from a route action so that it's obvious looking at the template what the point of the test is.